l10n_nl_xaf_auditfile_export/data/XmlAuditfileFinancieel3.2.xsd

```diff
@@ -653,7 +653,6 @@
 			<xsd:enumeration value="AT"/>
 			<xsd:enumeration value="AU"/>
 			<xsd:enumeration value="AW"/>
-			<xsd:enumeration value="AX"/>

```

Comment:
@Hayyan-Ebrahem As another example I will analyse this:
1. Look at the [most recent version of this file on the 8.0 branch in the original OCA repo](https://github.com/OCA/l10n-netherlands/blob/8.0/l10n_nl_xaf_auditfile_export/data/XmlAuditfileFinancieel3.2.xsd), and activate the [Blame view](https://github.com/OCA/l10n-netherlands/blame/8.0/l10n_nl_xaf_auditfile_export/data/XmlAuditfileFinancieel3.2.xsd).
2. See that `AX` was [added 4 months ago by Andrea in this commit](https://github.com/OCA/l10n-netherlands/commit/fa68f5d8dba46bf012925d4cd1cfce27fc2f9f45)
3. Conclude that reverting this should not be in this PR - it is not our work, but the reversion of a later update by someone from OCA.
4. Remove it from your branch in a new commit